models/action.go

```diff
@@ -26,30 +26,31 @@ type ActionType int
 
 // Possible action types.
 const (
-	ActionCreateRepo         ActionType = iota + 1 // 1
-	ActionRenameRepo                               // 2
-	ActionStarRepo                                 // 3
-	ActionWatchRepo                                // 4
-	ActionCommitRepo                               // 5
-	ActionCreateIssue                              // 6
-	ActionCreatePullRequest                        // 7
-	ActionTransferRepo                             // 8
-	ActionPushTag                                  // 9
-	ActionCommentIssue                             // 10
-	ActionMergePullRequest                         // 11
-	ActionCloseIssue                               // 12
-	ActionReopenIssue                              // 13
-	ActionClosePullRequest                         // 14
-	ActionReopenPullRequest                        // 15
-	ActionDeleteTag                                // 16
-	ActionDeleteBranch                             // 17
-	ActionMirrorSyncPush                           // 18
-	ActionMirrorSyncCreate                         // 19
-	ActionMirrorSyncDelete                         // 20
-	ActionApprovePullRequest                       // 21
-	ActionRejectPullRequest                        // 22
-	ActionCommentPull                              // 23
-	ActionPublishRelease                           // 24
+	ActionCreateRepo          ActionType = iota + 1 // 1
+	ActionRenameRepo                                // 2
+	ActionStarRepo                                  // 3
+	ActionWatchRepo                                 // 4
+	ActionCommitRepo                                // 5
+	ActionCreateIssue                               // 6
+	ActionCreatePullRequest                         // 7
+	ActionTransferRepo                              // 8
+	ActionPushTag                                   // 9
+	ActionCommentIssue                              // 10
+	ActionMergePullRequest                          // 11
+	ActionCloseIssue                                // 12
+	ActionReopenIssue                               // 13
+	ActionClosePullRequest                          // 14
+	ActionReopenPullRequest                         // 15
+	ActionDeleteTag                                 // 16
+	ActionDeleteBranch                              // 17
+	ActionMirrorSyncPush                            // 18
+	ActionMirrorSyncCreate                          // 19
+	ActionMirrorSyncDelete                          // 20
+	ActionApprovePullRequest                        // 21
+	ActionRejectPullRequest                         // 22
+	ActionCommentPull                               // 23
+	ActionPublishRelease                            // 24
+	ActionPullReviewDismissed                       // 25
```

Comment:
Looks like line 37 to 52 doesn't change, but why is it on the diff?

Reply:
I also don't know why ...

Reply:
Go fmt has changed the spacing... It's fine ignore it we just have to accept that go fmt will change things.